resources/sdk/purecloudgo/templates/apiclient.mustache

```diff
@@ -93,7 +93,6 @@ func (c *APIClient) CallAPI(path string, method string,
 	// Set default headers
 	if c.configuration.DefaultHeader != nil {
 		for k, v := range c.configuration.DefaultHeader {
-			fmt.Printf("  %v=%v", k, v)
```

Comment:
Thanks, this shouldn't be in here. The intent for the SDKs to be silent under normal circumstances.